ansible/tasks/postgres-extensions/12-pljava.yml

```diff
@@ -9,19 +9,27 @@
     update_cache: yes
     install_recommends: no
 
-- name: pljava - download latest release
-  get_url:
-    url: https://github.com/tada/pljava/archive/V{{ pljava_release }}.tar.gz
-    dest: /tmp/pljava-{{ pljava_release }}.tar.gz
-    checksum: "{{ pljava_release_checksum }}"
-    timeout: 60
+#TODO: revert to using main repo after PG15 support is merged: https://github.com/tada/pljava/pull/413
+# - name: pljava - download latest release
+#   get_url:
+#     url: https://github.com/tada/pljava/archive/V{{ pljava_release }}.tar.gz
+#     dest: /tmp/pljava-{{ pljava_release }}.tar.gz
+#     checksum: "{{ pljava_release_checksum }}"
+#     timeout: 60
+
+# - name: pljava - unpack archive
+#   unarchive:
+#     remote_src: yes
+#     src: /tmp/pljava-{{ pljava_release }}.tar.gz
+#     dest: /tmp
+#   become: yes
 
-- name: pljava - unpack archive
-  unarchive:
-    remote_src: yes
-    src: /tmp/pljava-{{ pljava_release }}.tar.gz
-    dest: /tmp
+- name: pljava - download latest release
   become: yes
+  git:
+    repo: https://github.com/supabase/pljava.git
+    dest: /tmp/pljava-{{ pljava_release }}
+    version: "master"
```

Comment:
Revert to using proper pljava releases once pg15 support gets merged into main repo

Reply:
(ideally a specific commit sha, though I guess if we're controlling the branch it's not the end of the world)

Reply:
Very fair point - will address this together with the merge conflicts